miden-lib/src/tests/test_note.rs

```diff
@@ -354,9 +351,10 @@ fn test_note_script_and_note_args() {
         (tx_inputs.input_notes().get_note(1).note().id(), note_args[0]),
     ]);
 
-    let tx_args = TransactionArgs::new(None, Some(note_args_map));
+    let mut tx_args = TransactionArgs::new(None, Some(note_args_map), AdviceMap::default());
+    tx_args.get_advice_map_mut().extend(tx_args_notes.get_advice_map().clone());
```

Comment:
I see we are using the mutable reference to the advice map here - but could we not just pass `tx_args_notes.get_advice_map().clone()` as a construction parameter on line 354 instead of `AdviceMap::default()`?